ushahidi/platform-client
# SurveySDK: Add color to list of properties to return on getSurveys

This pull request makes the following changes:
- Fetched the survey-colour from the API request

Testing checklist:
- Make sure there is a post from a survey configured with a colour
- Go to map-view

- [ ] Look at the list to the left, the survey should have a colour-band on the left of the card.

Fixes [ushahidi/platform#4303](https://github.com/ushahidi/platform/issues/4303).

Ping @ushahidi/platform
